src/main/java/org/mastodon/mamut/MamutAppModel.java

```diff
@@ -157,4 +160,16 @@ public BranchGraphSynchronizer getBranchGraphSync()
 	{
 		return branchGraphSync;
 	}
+
+	public void close() {
```

Comment:
Could you please check, if you have the correct code style settings? The bracket for the method should be a new line.